libs/template/templates/dbt-sql/databricks_template_schema.json

```diff
@@ -0,0 +1,53 @@
+{
+    "welcome_message": "\nWelcome to the dbt template for Databricks Asset Bundles!",
+    "properties": {
+        "project_name": {
+            "type": "string",
+            "pattern": "^[A-Za-z_][A-Za-z0-9_]+$",
+            "pattern_match_failure_message": "Name must consist of letters, numbers, and underscores.",
+            "default": "dbt_project",
+            "description": "\nPlease provide a unique name for this project.\nproject_name",
+            "order": 1
+        },
+        "http_path": {
+            "type": "string",
+            "pattern": "^/sql/.\\../warehouses/[a-z0-9]+$",
```

Comment:
How does this pattern work without `+` or `*`?

Reply:
I'm not sure I understand this question. There is a `+` in there?

Reply:
Or maybe you're referring to the `\.\\..` part? That matches a version, like `1.0`.